pkgs/by-name/ci/ci-edit/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "ci-edit";
+  version = "unstable-2023-04-11";
```

If the official release (although not tagged) was v51, maybe we could make the version reflect that? https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

```suggestion
  version = "v51-unstable-2023-04-11";
```

---

applied without the v